### DISCUSSION TITLE
能用neovis添加、修改、删除节点或关系么？

### DISCUSSION THREAD

**Discussion by abruptxy** [neo4j-contrib/neovis.js#319] [Q&A]
能用neovis添加、修改、删除节点或关系么？
是否有具体的调用方法或示例？

多谢多谢！！！

**Comment by thebestnom** [comment]
English please 😅

**Reply by abruptxy** [reply]
Can I use neovis to add, modify or delete node directly？
Is there any specific example?
thanks!

**Reply by thebestnom** [reply]
Not really, that's not the usecase of neovis 😅 but if you want you can implement it with neo4j driver and neovis events

**Reply by liudaihai** [reply]
解决了吗哥们，我也想用这个做一个前端界面
